Packs/RsaNetWitnessPacketsAndLogs/ReleaseNotes/1_0_5.md

```diff
@@ -1,8 +1,12 @@
-
 #### Integrations
 
 ##### RSA NetWitness Packets and Logs
 
 - You can now use credentials when configuring the following fields:
   - *Username*
   - *Password*
```

Comment:
please merge from master, it is already supported in the previous version: 
https://github.com/demisto/content/pull/27372/files#diff-1ef814930bd2e017cb530a17d421dfde84875a06ed9aef7665d540e6b4e21a74

Reply:
done :)